card/mod/search/set/all/filter.rb

```diff
@@ -5,8 +5,8 @@ def filter_form data={}, form_args={}
     render_haml :filter_form, categories: data, form_args: form_args
   end
 
-  def view_template_path view
-    super(view, __FILE__)
+  def view_template_path view, set_path= __FILE__
+    super(view, set_path)
```

Comment:
This is in an `all` set, so overriding the other `view_template_path` without a path argument breaks a bunch of thins in wikirate.

I'm not loving this API, because there's no easy way for other `all` sets to use haml templates without overriding each other.  I think we need to look into the haml API, perhaps in keeping with your `@todo` comments in haml.rb.  More pair coding!